src/test/regress/sql/multi_insert_select.sql

```diff
@@ -1574,6 +1574,29 @@ INSERT INTO text_table (part_col) SELECT val FROM text_table;
 INSERT INTO text_table (part_col) SELECT val::text FROM text_table;
 insert into table_with_starts_with_defaults (b,c) select b,c FROM table_with_starts_with_defaults;
 
+-- Test on partition column without native hash function 
+CREATE TABLE test
+(
+    id BIGINT,
+    time DATE
+);
+
+CREATE TABLE test_second
+(
+    id BIGINT,
+    time DATE
+);
+
+SELECT create_distributed_table('test', 'time');
+SELECT create_distributed_table('test_second', 'time');
+
+INSERT INTO test VALUES(1, '11-11-1980');
+INSERT INTO test_second SELECT * FROM test;
```

Comment:
Maybe, we can use more meaningful names for temporary tables;

`test ` -> `raw_table`
`test_second ` -> `summary_table`

Then;
```sql
INSERT INTO summary_table SELECT time, count(*) FROM raw_table;
```